packages/patterns/src/components/pattern-convert-button.js

```diff
@@ -86,7 +86,10 @@ export default function PatternConvertButton( {
 				) &&
 				// Hide when current doesn't have permission to do that.
 				// Blocks refers to the wp_block post type, this checks the ability to create a post of that type.
-				!! canUser( 'create', 'blocks' );
+				!! canUser( 'create', {
+					kind: 'postType',
+					name: 'wp_block',
+				} );
```

Comment:
Tbh, I wish this API looks looked more similar to `getEntityRecord`, or the other way around. Then you could maybe also reuse the same args for fetching and checking permissions.

Reply:
I think under the hood on the REST API level it's exactly what happens as it sometimes has to call OPTIONS request so sharing the same list of params in that case was warranted when using `canUserEditEntityRecord`.

Reply:
Yeah, could we still change it, since `canUser` has only just been modified?

Reply:
Or should `getEntityRecord(s)` allow (clearer) named parameters? I guess it would be good to have more opinions here. Cc @jsnajdr @youknowriad

Reply:
If we use similar arguments, it would be hard to distinguish whether the user wants an entity-related or REST API resource query.

How should we handle cases like this without collision? While IDs are usually an `int`, this isn't a requirement.
```js
canUser( 'create', 'media' );
canUser( 'create', 'root', 'media' );
```

Reply:
Yeah, I think using "resource" in the first place was a wrong idea but it seems the ship has sailed here.

Reply:
Backward compatibility driven development is a factor here 😄

Reply:
Btw, the `block-directory` package has a valid use case for "resource".

https://github.com/WordPress/gutenberg/blob/1b7e3a7502693071e8877cc12f92a51f5a13212a/packages/block-directory/src/components/downloadable-blocks-panel/index.js#L29-L32